.github/workflows/build.yml

```diff
@@ -516,12 +520,12 @@ jobs:
           source ve1/bin/activate
           cd $INDEX_DIR
           update-index \
-            --index-branch=${INDEX_BRANCH} \
-            --index-file=${INDEX_FILE} \
-            --repository=${{ github.repository }} \
-            --chart-entry="${{ steps.prepare-chart-release.outputs.chart_entry }}" \
-            --chart-url="${{ steps.prepare-chart-release.outputs.chart_url }}" \
-            --version="${{ steps.prepare-chart-release.outputs.version }}"
+            --index-branch="$INDEX_BRANCH" \
+            --index-file="$INDEX_FILE" \
+            --repository="$GITHUB_REPOSITORY" \
+            --chart-entry="$CHART_ENTRY" \
+            --chart-url="$CHART_URL" \
+            --version="$VERSION"
```

Comment:
As a general practice, I would prefer if these should all be enclosed in brackets. E.g. `$INDEX_BRANCH` should be `${INDEX_BRANCH}` as it was before, and the remaining environment variables should match.

Good explanation here: https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables.

It's [my] preference and convention, not a requirement, and so I'll leave it to your discretion to address this feedback given that this case isn't affected by the lack of curly braces.